dscensor/compose.dev.yaml

```diff
@@ -0,0 +1,9 @@
+services:
+  dscensor:
+    build:
+      context: .
+    container_name: dscensor_openapi_dev
```

Comment:
"openapi" shouldn't be in the container name since the use of OpenAPI is just an implementation detail.